<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasterNeighborhoodStatisticsClass</class>
 <widget class="QWidget" name="RasterNeighborhoodStatisticsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RasterNeighborhoodStatistics</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLineEdit" name="lineEditGrid">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输入栅格：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/openFile.png</normaloff>:/login/res/openFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="begin">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>380</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditWholeCSVResult">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出表格（不同波段）：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveMaxFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMaxResult">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格（最大值）：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMeanResult">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格（均值）：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMinResult">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出栅格（最小值）：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveMinFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="saveMeanFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="saveWholeCSVFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;选择相应的路径即可得到所需的结果&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;（对单位栅格分析后生成表格较慢，请慎选~）&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCSVResult">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>输出表格（单位栅格）：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveCSVFile">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>330</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="login.qrc">
        <normaloff>:/login/res/saveFile.png</normaloff>:/login/res/saveFile.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="login.qrc"/>
 </resources>
 <connections/>
</ui>
